gradle.properties

```diff
@@ -1,26 +1,9 @@
 sovityEdcExtensionsVersion=0.0.1-SNAPSHOT
 sovityEdcExtensionGroup=de.sovity.edc.ext
+sovityBrokerServerGroup=de.sovity.broker
 sovityEdcGroup=de.sovity.edc
 edcGroup=org.eclipse.edc
 edcVersion=0.2.1.2
```

Comment:
can't this also be part of the libs.versions.toml?

Reply:
sure